annotation_widgets/event_validation/gui.py

```diff
@@ -0,0 +1,183 @@
+import tkinter as tk
+from collections import OrderedDict
+from tkinter import font, ttk
+
+from .logic import EventValidationLogic
+
+
+class EventValidationStatusBar(tk.Frame):
```

Comment:
Decouple Logic and StatusBar classes. Logic should not be passed to the StatusBar as an argument.
Add `get_status_data_callback: Callable` argument to the StatusBar constructor, and fetch status data in the `update_status` method using this callback

Reply:
Logic and StatusBar were separated.